src/current/cockroachcloud/advanced-cluster-management.md

```diff
@@ -142,17 +142,19 @@ Setting maintenance windows requires the [Cluster Admin]({% link cockroachcloud/
 Maintenance operations that are critical for cluster security or stability may be applied outside of the maintenance window, and upgrades that begin in a maintenance window may not always be completed by the end of the window.
 {{site.data.alerts.end}}
 
-To set a maintenance window:
+To set a maintenance window in the {{ site.data.products.cloud }} console:
 
 1. Click the pencil icon next to **Cluster maintenance** to edit the maintenance window.
 1. From the **Day** dropdown, select the day of the week during which maintenance may be applied.
 1. From the **Start of window** dropdown, select a start time for your maintenance window in UTC.
 
     The window will last for 6 hours from the start time.
 
-1. (Optional) If you want to delay automatic patch upgrades for 60 days, switch **Delay patch upgrades** to **On**.
+1. (Optional) If you want to delay automatic patch upgrades, switch **Delay patch upgrades** to **On**. You can set a deferral period of 30, 60, or 90 days after the patch is released.
 
-    Enable this setting for production clusters to ensure that development and testing clusters are upgraded before production clusters. This setting applies only to patch versions and not to other kinds of upgrades.
+    Enable this setting for production clusters to ensure that development and testing clusters are upgraded before production clusters. This setting applies only to patch versions and not to other kinds of upgrades. The patch upgrade occurs during a maintenance window after the deferral period.
```

Comment:
Can we be more specific about what "other kinds of upgrades" means? It's not intuitive to me what the alternatives to patch upgrades are.

Also, should it be "patch upgrades" for clarity?